contrib/format-maprdb/pom.xml

```diff
@@ -75,6 +75,12 @@
     <dependency>
       <groupId>com.mapr.fs</groupId>
       <artifactId>mapr-hbase</artifactId>
+      <exclusions>
+        <exclusion>
+          <artifactId>commons-codec</artifactId>
+          <groupId>commons-codec</groupId>
+        </exclusion>
+      </exclusions>
```

Comment:
I think it would be better to create another Jira and move these changes in the pom files there since it does not connect with the issue described in DRILL-5717.